zict/buffer.py

```diff
@@ -58,8 +58,13 @@ def __init__(
 
     def fast_to_slow(self, key, value):
         self.slow[key] = value
-        for cb in self.fast_to_slow_callbacks:
-            cb(key, value)
+        try:
+            for cb in self.fast_to_slow_callbacks:
+                cb(key, value)
+        # LRU catches exception, raises and makes sure keys are not lost and located in fast.
```

Comment:
Thanks for working on this @ncclementi and reviewing @crusaderky. @ncclementi can you add a short description of this behavior change to the `Buffer` and `LRU` docstrings? For both classes, a description along the lines of "If an exception occurs during X then Y happens." in the appropriate parameter description (e.g. `on_evict` for `LRU`) should be sufficient

Reply:
I added information about what happens when an exception is raised in both `LRU` and `Buffer`. I think keeping the description of what happens in the core of the docstring is more informative than just in the `on_evict` argument. Let me know what you think, I can always move it.